cupy_backends/cuda/libs/cusparselt.pyx

```diff
@@ -460,47 +507,49 @@ cpdef spMMAPruneCheck2(Handle handle, MatDescriptor sparseMatDescr, isSparseA,
         <int*> d_valid, <runtime.Stream> stream)
     check_status(status)
 
-cpdef size_t spMMACompressedSize(Handle handle, MatmulPlan plan):
+cpdef spMMACompressedSize(Handle handle, MatmulPlan plan,
+                          size_t compressedSize, size_t compressBufferSize):
```

Comment:
this is also wrong, the sizes are inputs, they are outputs. This function should return a tuple with `compressedSize` and `compressedBufferSize`. 

https://docs.nvidia.com/cuda/cusparselt/functions.html#cusparseltspmmacompressedsize

compare with my changes in https://github.com/cupy/cupy/pull/8253/files#diff-87d4e531dae57eb73f43825516aa473a758c2854413136b71ee86ed783023a0dR486